pkg/project/auth/cache.go

```diff
@@ -190,11 +189,11 @@ func NewAuthorizationCache(reviewer Reviewer, namespaceInterface kcoreclient.Nam
 func (ac *AuthorizationCache) Run(period time.Duration) {
 
 	namespaceReflector := cache.NewReflector(
-		&controller.InternalListWatch{
```

Comment:
I guess I'll come back in later and replace standalone reflectors with shared informers where possible...